new-frontiers-14/frontier-station-14
# PTech Changes: Infinite ID cards and Encryption keys

## About the PR


In this PR I changed the items of the PTech vending machines to be infinite, there where it made sense. So that is everything except command items, items that are exclusive to a certain role and items that should be obtained from other places like the bureaucracy crate.

This PR also increases the amount of items sold of the following items:
The STC's stamp (was 1, now 3)
Command encryption key (was 3, now 10)
and it adds the Public Affair Liason stamp to the NFSD PTech.

## Why / Balance

I mostly changed this because it made sense.
If anything should be infinite, it should be the encryption keys and ID's from the SR's office. I don't think the PTech runs out that often, but it should be infinite just in case. The same goes for the badges, this would allow the sherrif to more freely hire from outside NFSD and still grant them badges. 
The number increase on the STC's stamp and the command encryption keys was done for the same reason. While I don't expect it to be needed, it doesn't hurt to plan ahead.
The PAL stamp was added because the STC's stamp is already in the normal PTech, and I feel like the PAL's stamp should also be able to be obtained in the same way.

## How to test

Spawn a PTech
See that some of the items are infinite
Spawn an NFSD PTech
Notice the same thing. 

## Media

![image](https://github.com/user-attachments/assets/68991922-b555-4de9-bed9-aa061d3f1a6c)
![image](https://github.com/user-attachments/assets/aceff5ef-1c23-4b45-92a1-0f05cadd640d)
![image](https://github.com/user-attachments/assets/391a0471-6021-4a2e-9e49-a69a0dd83c7f)
![image](https://github.com/user-attachments/assets/91bc0b11-0f03-4158-b366-f44d24491dc9)

## Requirements


## Breaking changes


**Changelog**



- tweak: The SR's and Sherrif's PTech vending machines have now been updated.
- add: The Public Affair Liason Stamp can now be found in the NFSD PTech.
